pcsx2/gui/Panels/BiosSelectorPanel.cpp

```diff
@@ -170,12 +170,13 @@ bool Panels::BiosSelectorPanel::ValidateEnumerationStatus()
 
 void Panels::BiosSelectorPanel::EnumThread::ExecuteTaskInThread()
 {
+	u32 region, version;
+	std::string description, zone;
```

Comment:
Were those part of a class and you made them locals? Hard to tell from the diff.

Reply:
They were globals before, but this is for scanning, so don't want to screw with global state.